photon-client/src/main/scala/com/linkedin/photon/ml/photon_io/README.md

```diff
@@ -0,0 +1 @@
+This package is slated for removal when photon disappears (and photon models are handled as Game fixed effect models).
```

Comment:
Photon is not disappearing. :) The the legacy GLMSuite, Driver, and related code is.